src/main/resources/dbreplica/migration/V1.0.10__add_missing_columns_plsql.sql

```diff
@@ -1,21 +1,21 @@
- ALTER TABLE task_history ADD COLUMN description Text;
- ALTER TABLE task_history ADD COLUMN notes JSONB;
- ALTER TABLE task_history ADD COLUMN region_name Text;
- ALTER TABLE task_history ADD COLUMN location_name Text;
- ALTER TABLE task_history ADD COLUMN additional_properties JSONB;
- ALTER TABLE task_history ADD COLUMN reconfigure_request_time Timestamp Without Time Zone;
- ALTER TABLE task_history ADD COLUMN next_hearing_id Text;
- ALTER TABLE task_history ADD COLUMN next_hearing_date Timestamp Without Time Zone;
- ALTER TABLE task_history ADD COLUMN priority_date Timestamp Without Time Zone NOT NULL;
- ALTER TABLE task_history ADD COLUMN last_reconfiguration_time Timestamp Without Time Zone;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN description Text;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN notes JSONB;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN region_name Text;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN location_name Text;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN additional_properties JSONB;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN reconfigure_request_time Timestamp Without Time Zone;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN next_hearing_id Text;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN next_hearing_date Timestamp Without Time Zone;
+ ALTER TABLE cft_task_db.task_history ADD COLUMN priority_date Timestamp Without Time Zone NOT NULL;
```

Comment:
Will this `NOT NULL` work?  In AAT there will be existing records so you'd have to state what to do with them

Reply:
Yes Agreed. Removed NOT NULL